### PR TITLE
Update for the 2021 (v200) WorldCover map release

### PR DESCRIPTION
**Pull request by dzanaga**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/esa-worldcover.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-Documentation: Documentation is available [here](https://esa-worldcover.s3.amazonaws.com/readme.html).
+Documentation: Documentation is available [here](https://esa-worldcover.org/en/data-access).
 Contact: https://esa-worldcover.org/en/contact
-ManagedBy: "[Sinergise on behalf of VITO](https://vito.be)"
-UpdateFrequency: Not yet defined.
+ManagedBy: "[VITO](https://vito.be)"
+UpdateFrequency: Yerly.
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:7]: this looks like a typo

File: datasets/esa-worldcover.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+Description: The European Space Agency (ESA) WorldCover product provides global land cover maps for 2020 & 2021 at 10 m resolution based on Copernicus Sentinel-1 and Sentinel-2 data. The WorldCover product comes with 11 land cover classes and has been generated in the framework of the ESA WorldCover project, part of the 5th Earth Observation Envelope Programme (EOEP-5) of the European Space Agency. A first version of the product (v100), containing the 2020 map was released in October 2021. The 2021 map was released in October 2022 using an improved algorithm (v200). The WorldCover 2020 and 2021 maps were generated with different algorithm versions and therefore changes between the maps should be treated with caution as these contain both real land cover changes as well as changes due to the used algorithms.
 
-Documentation: Documentation is available [here](https://esa-worldcover.s3.amazonaws.com/readme.html).
+Documentation: Documentation is available [here](https://esa-worldcover.org/en/data-access).
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:4]: The documentation (link)[https://esa-worldcover.org/en/data-access] is not public. Appearing as 403, access denied

File: datasets/esa-worldcover.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 Contact: https://esa-worldcover.org/en/contact
-ManagedBy: "[Sinergise on behalf of VITO](https://vito.be)"
-UpdateFrequency: Not yet defined.
+ManagedBy: "[VITO](https://vito.be)"
+UpdateFrequency: Yerly.
 Collabs:
   ASDI:
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:9]: @apprivet do you approve? It's the same as https://registry.opendata.aws/esa-worldcover/ plus 2021.

File: datasets/esa-worldcover.yaml [hunk 4]
```diff
@@ -21,6 +21,7 @@ Tags:
   - disaster response
   - mapping
   - synthetic aperture radar
+  - land cover
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:24]: @dzanaga can you add the same tags from https://registry.opendata.aws/esa-worldcover/?

File: datasets/esa-worldcover.yaml [hunk 5]
```diff
@@ -29,25 +30,48 @@ Resources:
     Region: eu-central-1
     Type: S3 Bucket
     RequesterPays: False
+    Explore:
+    - '[STAC endpoint](https://services.terrascope.be/stac/)'
 DataAtWork:
   Tutorials:
+  - Title: Exploring the dataset (STAC API examples)
+    URL: https://github.com/ESA-WorldCover/esa-worldcover-datasets
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:38]: @dzanaga can you make the github repo public if you want to use it for a documentation link?

File: datasets/esa-worldcover.yaml [hunk 6]
```diff
@@ -29,25 +30,48 @@ Resources:
...
+  - Title: WorldCover GEE App
+    URL: https://vitorsveg.users.earthengine.app/view/worldcover
+    AuthorName: VITO
+    AuthorURL: https://esa-worldcover.org  
   Publications:
+  - Title: ESA WorldCover 10 m 2020 v100
+    URL: https://doi.org/10.5281/zenodo.5571936
+    AuthorName: Zanaga, D., Van De Kerchove, R., De Keersmaecker, W., Souverijns, N., Brockmann, C., Quast, R., Wevers, J., Grosu, A., Paccini, A., Vergnaud, S., Cartus, O., Santoro, M., Fritz, S., Georgieva, I., Lesiv, M., Carter, S., Herold, M., Li, Linlin, Tsendbazar, N.E., Ramoino, F., Arino, O., 2021.
+  - Title: ESA WorldCover 10 m 2021 v200
+    URL: https://doi.org/10.5281/zenodo.7254221
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:67]: @dzanaga this url results in error "This DOI cannot be found in the DOI System..."

**Comment by jeffejefe** [comment]
@dzanaga thanks for getting this yaml ready. I added comments above. If you make your branch editable I can commit directly into that if I find any formatting issues. For now I would try to fix above and then we'll check if the tests build.

File: datasets/esa-worldcover.yaml (hunk 1 above)
Comment (jeffejefe) [reply datasets/esa-worldcover.yaml:7]: should be "yearly"

File: datasets/esa-worldcover.yaml [hunk 7]
```diff
@@ -21,6 +21,7 @@ Tags:
   - disaster response
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:21]: you can add the `-stac` tag

File: datasets/esa-worldcover.yaml [hunk 8]
```diff
@@ -29,25 +30,48 @@ Resources:
     Region: eu-central-1
     Type: S3 Bucket
     RequesterPays: False
+    Explore:
+    - '[STAC endpoint](https://services.terrascope.be/stac/)'
```
Comment (jeffejefe) [inline datasets/esa-worldcover.yaml:34]: I noticed this didn't work with STAC Browser due to missing headers: https://radiantearth.github.io/stac-browser/#/external/services.terrascope.be/stac/

This doc may have some useful info: https://www.stackhawk.com/blog/fixing-no-access-control-allow-origin-header-present/.

Comment (dzanaga) [reply datasets/esa-worldcover.yaml:34]: Hi Mike, thanks. We need to look into that. Right now it's using our terrascope.be STAC API, maybe that's not fully compliant, we'll need to see how to update that, but we have a notebook that uses it.
The github repo is now public.

File: datasets/esa-worldcover.yaml (hunk 6 above)
Comment (dzanaga) [reply datasets/esa-worldcover.yaml:67]: It's pubished now.

File: datasets/esa-worldcover.yaml (hunk 5 above)
Comment (dzanaga) [reply datasets/esa-worldcover.yaml:38]: Done

File: datasets/esa-worldcover.yaml (hunk 2 above)
Comment (dzanaga) [reply datasets/esa-worldcover.yaml:4]: Temporarly offline, it will be on at 3PM CET

File: datasets/esa-worldcover.yaml (hunk 1 above)
Comment (dzanaga) [reply datasets/esa-worldcover.yaml:7]: thanks

**Comment by jeffejefe** [comment]
@dzanaga  Closing this PR as the dataset is live: https://registry.opendata.aws/esa-worldcover-vito/

**Pull request closed without merging**